flux-manifests/kyverno-policies-connectivity.yaml

```diff
@@ -0,0 +1,13 @@
+api_version: generators.giantswarm.io/v1
+app_catalog: control-plane-catalog
+app_destination_namespace: giantswarm
+app_name: kyverno-policies-observability
```

Comment:
should all instances of `observability` not be changed to `connectivity`?

Reply:
:see_no_evil: expert tip: safe the file after changing the content :see_no_evil:

Reply:
:sweat_smile: